homeassistant/components/filter/sensor.py

```diff
@@ -149,6 +150,7 @@
             cv.entity_domain(INPUT_NUMBER_DOMAIN),
         ),
         vol.Optional(CONF_NAME): cv.string,
+        vol.Optional(CONF_UNIQUE_ID): cv.string,
```

Comment:
This violates [ADR0007](https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md)

Reply:
hmmm didn't consider that, but I understand

FYI recently added https://github.com/home-assistant/core/pull/59205 and https://github.com/home-assistant/core/pull/58596 face the same issue and are present in the current beta.

Reply:
K... looking at #58596 it seems this is allowed...

Reply:
I have opened a PR to amend the ADR.